sensorhub-core/src/main/java/org/sensorhub/impl/system/SystemDriverTransactionHandler.java

```diff
@@ -477,12 +479,12 @@ public void handleEvent(Event e)
                     update(driver);
                 }
                 
-                else if ((e instanceof SystemChangedEvent) && driver instanceof ISystemGroupDriver)
-                {
-                    var memberProc = ((ISystemGroupDriver<?>) driver).getMembers().get(eventUid);
-                    if (memberProc != null)
-                        registerMember(memberProc);
-                }
+//                else if ((e instanceof SystemChangedEvent) && driver instanceof ISystemGroupDriver)
+//                {
+//                    var memberProc = ((ISystemGroupDriver<?>) driver).getMembers().get(eventUid);
+//                    if (memberProc != null)
+//                        registerMember(memberProc);
+//                }
             }
```

Comment:
Can you remove the commented section if it's not needed anymore?
We have it in history anyway if we ever need to revert